bench/axis_opencode_native_fallback.go

```
@@ -0,0 +1,254 @@
+			{Name: "fixture: repository", Fixture: baseRepo},
+			{Name: "sync persists and routes native fallback assignments", Skip: pinnedOpenCodeUnavailable, Composite: syncNativeFallbackAssignments},
+		},
+	}}
+}
+
+func pinnedOpenCodeUnavailable(*Sandbox) string {
+	path, err := exec.LookPath("opencode")
+	if err != nil {
+		return "opencode is unavailable"
+	}
+	output, err := exec.Command(path, "--version").CombinedOutput()
```

golangci-lint (2.12.2)

[medium] G204: Subprocess launched with variable

(gosec)